packages/@o3r/eslint-config/schematics/ng-add/eslint/templates/project/eslint.local.config.__extension__.template

```diff
@@ -1,19 +1,27 @@
-<% if (extension === 'mjs') { %>import { dirname } from 'node:path';
+<% if (extension === 'mjs') {%>import { dirname } from 'node:path';
 import { fileURLToPath } from 'node:url';
-
+<% if (isApp) {
+  %>import globals from 'globals';<%
+} %>
 const __filename = fileURLToPath(import.meta.url);
 // __dirname is not defined in ES module scope
 const __dirname = dirname(__filename);
-
-<% } %><%= extension === 'mjs' ? 'export default' : 'module.exports =' %> [
+<% } else if (isApp) { %>const globals = require('globals');
```

Comment:
looks like the same condition as above no? `if (isApp)`
and I'm not sure if there's a problem with the double closing bracket `}` from lines 5 and 9

Reply:
If you remove the template string you will have 
```ts
if (extension === 'mjs') {
  if (isApp) {
    ...
  }
} else if (isApp) {
  ...
}
```
so one is for `mjs`, the other one for `cjs` or `js` extension.